Kentico.Kontent.Delivery/Helpers/DeliveryOptionsHelpers.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Kentico.Kontent.Delivery.Abstractions;
+using Kentico.Kontent.Delivery.Configuration;
+using System;
+
+namespace Kentico.Kontent.Delivery.Helpers
+{
+    /// <summary>
+    /// A class which contains helper methods on <see cref="DeliveryOptions"/>.
+    /// </summary>
+    public static class DeliveryOptionsHelpers
+    {
+        /// <summary>
+        /// Builds the <see cref="DeliveryOptions"/> instance from the delegate.
+        /// </summary>
+        /// <param name="buildDeliveryOptions"></param>
+        /// <returns></returns>
+        public static DeliveryOptions Build(Func<IDeliveryOptionsBuilder, DeliveryOptions> buildDeliveryOptions)
```

Comment:
Can't we just somehow abstract this and retrieve the `IDeliveryOptionsBuilder`?

Reply:
This is just a helper that evaluates `IDeliveryOptionsBuilder`.